### DISCUSSION TITLE
Embed Plausible report on Notion

### DISCUSSION THREAD

**Discussion by emilienbidet** [plausible/analytics#2377] [Anything Else]
Hello Plausible team !

I would like to embed my analytics report on Notion.
I tried by using the iframe in a Apption.co widget but it doesn't works.

Does anyone have an idea ?
Thank you.

**Comment by bogplau** [comment]
Hi Emilien, 

I've tested the Apption.co widget and you're right it doesn't work. However you don't need it, this is how you embed your dashboard into notion: 

1. Create [a shared link](https://plausible.io/docs/embed-dashboard) ( NOTE: do not set a password )
2. Copy the newly created link. ( i.e. https://plausible.io/share/YourWebsite.com?auth=xxxx )
3. In Notion, type `/embed` and choose the first option ( embed for PDFs, Google Maps and more )

![plausible_notion_1](https://user-images.githubusercontent.com/115988568/197971761-e4adcb61-7804-4e8f-a760-23d462235535.png)

4. Paste the share linked from step 2. 

![plausible_notion](https://user-images.githubusercontent.com/115988568/197971953-b1f7ae2e-afdc-4e73-84fe-3cb6a9abb72a.png)


Hope this helps!


**LE:** it actually works with the iframe embed code as well.

**Comment by NathanC** [comment]
@bogplau thank you!

Do you know if there is a way to increase the embed height past the max height it seems to cap at? It requires scrolling within the embed, whereas I'd like to make the embed sufficiently tall to capture all of it. Any ideas?

**Comment by bogplau** [comment]
No worries @NathanC, glad I could help! :)

To resize it, just hover your mouse cursor over the embed and drag to adjust to the desired size. 

![resize](https://user-images.githubusercontent.com/115988568/198656097-27a27b97-31b0-428f-aff1-999410f4ff15.png)

**Reply by NathanC** [reply]
Are you able to get it to the full height? I'm not-- I can increase the hight, but then it hits a hard limit.

**Reply by bogplau** [reply]
oh, you're right. sorry about that :( , thought you can go full height with it. I'm not sure how to resize a block in notion.

**Reply by NathanC** [reply]
No worries! It's still useful, just would be a bit cleaner if we were able to get around vertical scroll. This is specially a notion thing though, I might make a feature request to notion in a bit.
